src/actions/actions.js

```diff
@@ -0,0 +1,160 @@
+import axios from 'axios';
```

Comment:
Use `apisauce` intead of `axios`

It wraps around axios, adding more functionalities